test/converter-test.js

```diff
@@ -160,6 +160,23 @@ describe('Converter', () => {
       });
     });
 
+    context('ozt', () => {
+      it('should convert', () => {
+        testConvert(
+          [
+            "1.25 troy ounces",
+            "1.25 oz t",
+            "5 ozt"
+          ],
+          {
+            "1.25 troy ounces" : "38.88 g",
+            "1.25 troy ounces" : "38.88 g",
```

Comment:
Oh, one more thing I noticed (sorry for all the back-and-forth, I should have caught this earlier) is that these two keys are the same, so the bottom one will override the top one. You can see what I mean if you change line 172 to say `"1.25 troy ounces" : "0 g",` and the test will still pass

Reply:
Updated.